internal/pkg/mapper/orchestrator/local_save.go

```diff
@@ -23,6 +23,7 @@ func (m *orchestratorMapper) MapBeforeLocalSave(recipe *model.LocalSaveRecipe) e
 		LocalSaveRecipe: recipe,
 		logger:          m.logger,
 		config:          recipe.Object.(*model.Config),
+		configPath:      recipe.ObjectManifest.GetPathInProject(),
```

Comment:
Vsimol som si, ze do testu `template-create` som zabudol pridat orchestator.
V tomto PR ho pridavam + fixujem drobny bug.

V konfiguracii orchestratora na disku je path, ku `target config`, ktory sa spusta.
Takto path sa hladala v adresary `branche` (parent configu orchestratora).

Orchestrator v sablone ale nema parent branch.
A preto to v sablone nefunguje.

Riesenie je hladat `target config` v `orchestrator config.ParentPath()`.
V projekte to bude `branch`, v sablone to bude `(empty string)`.